app/models/page.rb

```diff
@@ -0,0 +1,23 @@
+# == Schema Information
+#
+# Table name: pages
+#
+#  id           :integer          not null, primary key
+#  page_type    :integer
+#  title        :string
+#  content_html :string
+#  content      :string
+#  source_url   :string
+#  status       :integer
+#  screenshot   :string
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#  company_id   :integer
+#
+
+class Page < ApplicationRecord
+  STATUSES = %i[active pending finished].freeze
+  PAGE_TYPES = %i[bing angle.co linkedin crunchbase].freeze
+  enum status: STATUSES
```

Comment:
remove ENUM. Just use string column value. It's easier to debug in future and work with such.
In general enums are good only for columns like "male/female".
For statuses, page_type - use STRING column